### PR TITLE
docs(combobox): increase chromatic coverage of variants

### PR DESCRIPTION
**Pull request by mdt2**
## Description

Increase Chromatic coverage for Combobox variants.

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

### Validation steps @jenndiaz 
- [x] [Combobox in Storybook](https://pr-2481--spectrum-css.netlify.app/preview/?path=/story/components-combobox--default) should look the same (other than the addition of the Quiet story) since these are Chromatic-only changes 
- [x] In `combobox.stories.js`, comment out the isChromatic import and add this line to force the Chromatic view to show `const isChromatic = () => true;`. Then open Storybook and check that:
    - [x] all default variants are represented for the [Default story](https://pr-2481--spectrum-css.netlify.app/preview/?path=/story/components-combobox--default)
    - [x] all quiet variants are represented in the [Quiet story](https://pr-2481--spectrum-css.netlify.app/preview/?path=/story/components-combobox--quiet)

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:

- [ ] The pages render correctly, are accessible, and are responsive.

2. If components have been modified, VRTs have been run on this branch:

- [ ] VRTs have been run and looked at.
- [ ] Any VRT changes have been accepted (by reviewer and/or PR author), or there are no changes.

## To-do list

- [ ] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [ ] I have updated relevant storybook stories and templates.
- [ ] I have tested these changes in Windows High Contrast mode.
- [ ] If my change impacts **other components**, I have tested to make sure they don't break.
- [ ] If my change impacts **documentation**, I have updated the documentation accordingly.
- [ ] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

**Comment by jenndiaz** [comment]
Do we want to also account for side labels with ComboBox? 
https://github.com/adobe/spectrum-css/pull/2483#event-11689403439

**Comment by jenndiaz** [comment]
The verification steps mention storybook being the same but I am noticing we added a quiet story, just wanted to confirm that was intentionally added and that the verification steps are just out of date.

File: components/combobox/stories/combobox.stories.js [hunk 1]
```diff
@@ -127,7 +141,34 @@ export default {
...
 		isDisabled: false,
 		showFieldLabel: false,
-		fieldLabelText: "Select location"
+		fieldLabelText: "Select location",
+		content: [
+			Menu({
+				role: "listbox",
+				subrole: "option",
+				isSelectable: true,
+				items: [
```
Comment (jenndiaz) [inline components/combobox/stories/combobox.stories.js:150]: We should also pass `size` to menu so it changes when we toggle the control for combobox

File: components/combobox/stories/combobox.stories.js [hunk 2]
```diff
@@ -141,67 +182,84 @@ export default {
...
+	<div style=${args.isOpen && "padding-bottom: 10rem;"}>
+		${Template({
+			...args,
+			isLoading: true,
+		})}
+	</div>
+	<div style=${args.isOpen && "padding-bottom: 10rem;"}>
+		${Template({
+			...args,
+			isValid: true,
```
Comment (jenndiaz) [inline components/combobox/stories/combobox.stories.js:218]: I don't believe there should be a valid state for Combobox

File: components/combobox/stories/combobox.stories.js [hunk 3]
```diff
@@ -141,67 +182,84 @@ export default {
...
+		${Template({
+			...args,
+			showFieldLabel: true,
+			fieldLabelText: "Select location, this label should wrap",
+			fieldLabelPosition: "left",
+		})}
+	</div>
+`;
+
+const closedVariants = (args) => defaultVariants({...args, isOpen: false});
```
Comment (jenndiaz) [inline components/combobox/stories/combobox.stories.js:244]: ohh this is smart.

File: components/combobox/stories/combobox.stories.js (hunk 2 above)
Comment (mdt2) [reply components/combobox/stories/combobox.stories.js:218]: I think you're right. I'm not sure why we had that as an option in Storybook! I'll remove.

File: components/combobox/stories/combobox.stories.js [hunk 4]
```diff
@@ -113,6 +116,17 @@ export default {
...
+		fieldLabelPosition: {
+			name: "Field label position",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			options: ["top", "left"],
+			control: "select",
+			if: { arg: "showFieldLabel", truthy: true },
```
Comment (castastrophe) [inline components/combobox/stories/combobox.stories.js:116]: Nice, great touch here. Makes the UX so much nicer for visitors

**Review by castastrophe — APPROVED** [review]
A few suggestions but otherwise the code looks good!

**Pull request merged**